range_queries/sparse_table.cpp

```diff
@@ -0,0 +1,71 @@
+/*
+Implementation of Sparse Table
+
+Running Time Complexity
+Build : O(NlogN)
+Range Query : O(1)
+*/
+
+
+#include<vector>
+#include<iostream>
+#include<algorithm>
+
+
+/**
+ * Util function to store pre-computed logs
+ */
+std::vector<int> computeLogs(int n) {
+    std::vector<int> logs(n+5);
```

Comment:
Shouldn't this be a paremeter?
Same in other parts of the code.

Reply:
Since this variable is getting modified in this function, I created a variable in this function and returned to the main function for future reference. I don't see any issue with this, just implemented differently.